acceptance/lib/helper.rb

```diff
@@ -1,4 +1,6 @@
 require 'beaker/dsl/install_utils'
+require 'puppetserver/acceptance/compat_utils'
+require 'puppetserver/acceptance/gem_utils'
```

Comment:
My preference would be to just put the `gem_utils` require statement back into the `.../subcommand/gem.rb` since that's what uses the one function in `gem_utils.rb`.  And the `compat_utils` one probably isn't strictly needed since the individual `puppet3_tests` that need it all require it on their own, right?